wagtail/coreutils.py

```diff
@@ -539,3 +540,14 @@ def _do_processing(self):
     def get_summary(self):
         opts = self.model._meta
         return f"{self.created_count}/{self.added_count} {opts.verbose_name_plural} were created successfully."
+
+
+def make_wagtail_template_fragment_key(fragment_name, page, site, vary_on=None):
```

Comment:
Given we're letting people customise the cache key components, do we need to account for that here/when mentioning this fn in docs?

Reply:
Ah, ignore me, just noticed the `cache_key` is already in here